ultravox/inference/infer.py

```diff
@@ -102,6 +159,9 @@ def _dataproc(self, sample: datasets.VoiceSample):
         inputs = {k: v.to(self.model.device) for k, v in inputs.items()}
         if "audio_values" in inputs:
             inputs["audio_values"] = inputs["audio_values"].to(dtype=self.dtype)
+        if batch:
+            for key, val in inputs.items():
+                inputs[key] = val.squeeze()
```

Comment:
nit: are you sure you want `squeeze()` or `squeeze(0)`?
Asking because the behaviour of `squeeze` is dynamic and can remove too many columns depending on the values.

Reply:
right, squeeze(0) is definitly correct here, thanks for the catch.